example/provider-local/garden/base/kustomization.yaml

```diff
@@ -8,6 +8,7 @@ resources:
 - secret-backup.yaml
 - secretbinding.yaml
 - https://raw.githubusercontent.com/gardener/gardener-extension-networking-calico/v1.31.1/example/controller-registration.yaml
+- https://raw.githubusercontent.com/gardener/gardener-extension-networking-cilium/v1.22.0/example/controller-registration.yaml
```

Comment:
We don't need the calico extension anymore now, right? Can you please remove it?

Reply:
Addressed with https://github.com/gardener/gardener/pull/7664/commits/57da282be02f4408a9c52b0680cb95ed3913048d.